BUILD

```diff
@@ -17,7 +17,7 @@ _local_environment(
     name="macos_local_env",
     compatible_platforms=["macos_arm64", "macos_x86_64"],
     # Avoid system Python interpreters, which tend to be broken on macOS.
-    python_interpreter_search_paths=["<PYENV>"],
+    python_bootstrap_search_path=["<PYENV>"],
```

Comment:
Should this be python_bootstrap__serch_path?

Reply:
Nope, per the previous design, we decided the name should resemble the envvar name, so that there are fewer special cases.